modules/labs-react/side-panel/lib/SidePanelToggleButton.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from 'react';
+import {
+  createElemPropsHook,
+  createSubcomponent,
+  ExtractProps,
+} from '@workday/canvas-kit-react/common';
+import {TertiaryButton} from '@workday/canvas-kit-react/button';
+import {transformationImportIcon} from '@workday/canvas-system-icons-web';
+import {Tooltip, TooltipProps} from '@workday/canvas-kit-react/tooltip';
+import {useSidePanelModel} from './useSidePanelModel';
+import {createStencil, handleCsProp} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
+
+export interface SidePanelToggleButtonProps extends ExtractProps<typeof TertiaryButton, never> {
+  /**
+   * The tooltip text to expand the side panel
+   */
+  tooltipTextExpand?: string;
+  /**
+   * The tooltip text to collapse the side panel
+   */
+  tooltipTextCollapse?: string;
+  tooltipProps?: Omit<TooltipProps, 'children'>;
+}
+
+export const sidePanelToggleButtonStencil = createStencil({
+  base: {
+    position: 'absolute',
+    top: system.space.x6,
+    width: system.space.x8,
+    insetInlineEnd: system.space.x4,
+  },
+  modifiers: {
+    state: {
+      collapsing: {
+        margin: 0,
+        transform: `scaleX(1)`,
+        ':dir(rtl)': {
+          transform: `scaleX(-1)`,
```

Comment:
make transform as var? and remove modifiers?

Reply:
I kept it as a modifier because it's a a prop for `origin` so it's easy to style, but if you feel strongly i can change it!